tests/test_laser_profiles.py

```diff
@@ -41,6 +46,92 @@ def gaussian():
     return profile
 
 
+def test_transverse_profiles_rt():
+    npoints = 4000
+    w0 = 10.0e-6
+    r = np.linspace(0, 12 * w0, npoints)
+
+    # GaussianTransverseProfile
+    print("GaussianTransverseProfile")
+    std_th = w0 / np.sqrt(2)
+    profile = GaussianTransverseProfile(w0)
+    field = profile.evaluate(r, np.zeros_like(r))
+    std = np.sqrt(np.average(r**2, weights=np.abs(field)))
+    print("std_th = ", std_th)
+    print("std = ", std)
+    assert np.abs(std - std_th) / std_th < 0.01
+
+    # LaguerreGaussianTransverseProfile
+    print("LaguerreGaussianTransverseProfile")
+    p = 2
+    m = 0
+    std_th = np.sqrt(5 / 2) * w0
+    profile = LaguerreGaussianTransverseProfile(w0, p, m)
+    field = profile.evaluate(r, np.zeros_like(r))
+    std = np.sqrt(np.average(r**2, weights=r * np.abs(field) ** 2))
```

Comment:
I modified this formula (`np.abs(field) -> r*np.abs(field)**2`) so that it has an easy analytical solution.